getting-started/getting-started-activiti-cloud/README.md

```diff
@@ -223,9 +223,9 @@ You are now ready to start consuming these services to automate your own busines
 
 Finally, you can access to all services Swagger documentation by pointing your browser to:
 
-* [http://activiti-cloud-gateway.EXTERNAL-IP.nip.io/rb-my-app/swagger-ui.html](http://activiti-cloud-gateway.external-ip.nip.io/rb-my-app/swagger-ui.html)
-* [http://activiti-cloud-gateway.EXTERNAL-IP.nip.io/audit/swagger-ui.html](http://activiti-cloud-gateway.external-ip.nip.io/audit/swagger-ui.html)
-* [http://activiti-cloud-gateway.EXTERNAL-IP.nip.io/query/swagger-ui.html](http://activiti-cloud-gateway.external-ip.nip.io/query/swagger-ui.html)
+* [http://activiti-cloud-gateway.EXTERNAL-IP.nip.io/rb-my-app/swagger-ui/index.html](http://activiti-cloud-gateway.external-ip.nip.io/rb-my-app/swagger-ui/index.html)
```

Comment:
The path was actually outdated as well. It should be `rb` instead of `rb-my-app`. Could you update it as well, please?

Reply:
Done